includes/discovery/sensors/temperature/bti800.inc.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
You should convert this file to the yaml base format,
See for example https://github.com/librenms/librenms/blob/941037e44e1afe492aecce8f939394782e3990a2/includes/definitions/discovery/fiberhome-switch.yaml#L16-L26

Reply:
I will test first on the server then I will post the output. Really appreciate your comment @Jellyfrog.